docs/code-howtos/testing.md

```diff
@@ -140,6 +140,16 @@ docker run -e MYSQL_ROOT_PASSWORD=root -e MYSQL_DATABASE=jabref -p 3800:3307 mys
 
 Set the environment variable `DBMS` to `mysql`.
 
+## Fetchers in tests
+
+Fetcher tests could be done by executing the Gradle task `fetcherTest`. This could be done by typing the following in the command line:
```

Comment:
```suggestion
Fetcher tests can be run locally by executing the Gradle task `fetcherTest`. This can be done by the following command in the command line:
```